tests/ClientAreaTest.php

```diff
@@ -107,6 +111,25 @@ public function testLoadServerPageServer(): void
                            ],
                        ],
                    ],
+                   'usage' => [
+                       'network' => [
+                           'incoming' => [
+                               'value' => 1024*1024*1024 / 2 - 1,
```

Comment:
You can use your new class Unit here

Reply:
Unit class has no constants for representing KiB, MiB... as bytes